data/brands/tourism/information.json

```diff
@@ -0,0 +1,24 @@
+{
+  "properties": {
+    "path": "brands/tourism/information",
+    "preserveTags": ["^name"],
+    "exclude": {"generic": ["^information$"]}
```

Comment:
This looks good to me 👍 
If you want to tell the build script not to bring in all the other `tourism=information` things that we've collected, you can add this line up here:
```js
    "skipCollection": true,
```